src/cmd/cli/command/commands.go

```diff
@@ -350,10 +352,6 @@ var RootCmd = &cobra.Command{
 			}
 		}
 
-		if isUpgradeCommand(cmd) {
```

Comment:
@commit111 Instead of checking whether it's the upgrade command, and then do something, I moved that `hideUpdate = true` line to the actual `upgradeCmd`